server/src/main/java/org/opensearch/index/merge/MergeStats.java

```diff
@@ -81,6 +84,12 @@ public MergeStats(StreamInput in) throws IOException {
         totalStoppedTimeInMillis = in.readVLong();
         totalThrottledTimeInMillis = in.readVLong();
         totalBytesPerSecAutoThrottle = in.readVLong();
+
+        if (in.getVersion().onOrAfter(Version.CURRENT)) {
+            cleanUpStats = in.readOptionalWriteable(UnreferencedFileCleanUpStats::new);
+        } else {
+            cleanUpStats = new UnreferencedFileCleanUpStats();
```

Comment:
This should be null - can you check again - in older versions there won't be this class

Reply:
Actually in older version, this code block itself will not be present.